### DISCUSSION TITLE
What is actions/checkout@v2 in Github action?

### DISCUSSION THREAD

**Discussion by karenworld** [community/community#25682] [Actions]
<p>Hi,<br>
I tried to search for an answer about ```<br>
actions/checkout@v2</p>
<pre><code class="lang-auto">
Can someone explains thing abit to me? Tks.</code></pre>

**Comment by mcaskill** [comment, marked as answer]
<p>It is an official GitHub Action used to check-out a repository so a workflow can access it.</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/actions/checkout">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/actions/checkout" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/29353/181092484-9337e31f-ef5c-4951-98e0-9dee4b6fc687.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/1/d148b0e43cf9831218afff6840576994119af4d5_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/d/1/d148b0e43cf9831218afff6840576994119af4d5_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/d/1/d148b0e43cf9831218afff6840576994119af4d5.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/1/d148b0e43cf9831218afff6840576994119af4d5_2_10x10.png"></div>

<h3><a href="https://github.com/actions/checkout" target="_blank" rel="noopener nofollow ugc">actions/checkout</a></h3>

  <p>Action for checking out a repo. Contribute to actions/checkout development by creating an account on GitHub.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>By default, this action will check-out to the SHA for that workflow’s event (such as <code>push</code> and <code>pull_request</code>). Otherwise, uses the default branch (usually <code>main</code> or <code>master</code> in a standard repository).</p>
